DOCS.md

```diff
@@ -28,7 +27,7 @@ The following is a sample Docker configuration in your .drone.yml file:
 publish:
   docker:
     username: kevinbacon
-    password: pa55word
+    password: $$DOCKER_PASSWORD
```

Comment:
I could see sneaking a link to the secrets docs up in the param list, or in a comment in the yaml example itself.